s3/pkg/datastore/yig/utils/global_id.go

```diff
@@ -0,0 +1,74 @@
+package utils
+
+import (
```

Comment:
since the idea is to generate global unique ids, should we make this a micro-service ?

Reply:
@rhsakarpos currently, yig storage driver needs this kind of function to generate the global unique id for multipart upload id. if other modules also need that function, we can discuss to make it as a micro-service, thanks.